plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftHiveMetastore.java

```diff
@@ -457,7 +457,7 @@ public PartitionStatistics getTableStatistics(HiveIdentity identity, Table table
                     .stopOn(MetaException.class, UnknownTableException.class, UnknownDBException.class)
                     .stopOnIllegalExceptions()
                     .run("getFields", stats.getGetFields().wrap(() -> {
-                        try (ThriftMetastoreClient client = createMetastoreClient()) {
+                        try (ThriftMetastoreClient client = createMetastoreClient(identity)) {
```

Comment:
1. commit title: "Avro tables" -> "Avro tables with schema url"
2. this can be tested if we run `TestAvroSchemaUrl` on a kerberized environment.

Reply:
Can you tell me exactly how to do 2?

Reply:
i see `io.trino.tests.product.hive.TestAvroSchemaUrl#testHiveCreatedTable` is already in STORAGE_FORMATS, so it's run with kerberized already. It's passing probably because the setup allows default Trino user to access the schema.